src/digitalIdentity-frontend/src/app/pages/change-password-page/change-password-page.component.html

```diff
@@ -0,0 +1,92 @@
+  <mat-card-header>
+    <mat-card-title class="change-passsword-card-title">
+      Changing Password
+    </mat-card-title>
+  </mat-card-header>
+
+  <mat-divider [inset]="true"></mat-divider>
+
+  <mat-card-content class="change-password-card-content">
+    <form class="change-password-form" [formGroup]="formGroup">
+      <!-- (ngSubmit)="changePasswordProcess()" -->
+      <mat-form-field>
+        <mat-label>Enter your email</mat-label>
+        <input
+          matInput
+          type="text"
+          placeholder="john.doe@example.org"
+          formControlName="email"
+        />
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Enter your old password</mat-label>
+        <input
+          matInput
+          [type]="hide_old ? 'password' : 'text'"
+          formControlName="old_password"
+        />
+        <mat-icon
+          matSuffix
+          (mousedown)="hide_old = !hide_old"
+          [attr.aria-label]="'Hide password'"
+          [attr.aria-pressed]="hide_old"
+        >
+          {{ hide_old ? "visibility_off" : "visibility" }}
+        </mat-icon>
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Enter your new password</mat-label>
+        <input
+          matInput
+          [type]="hide ? 'password' : 'text'"
+          formControlName="password"
+        />
+        <mat-icon
+          matSuffix
+          (click)="hide = !hide"
+          [attr.aria-label]="'Hide password'"
+          [attr.aria-pressed]="hide"
+        >
+          {{ hide ? "visibility_off" : "visibility" }}
+        </mat-icon>
+      </mat-form-field>
+      <div class="field-message" *ngIf="passwordStrengthError() == 'lengthError'">
+        Your password is too weak: It should contain at least 8 characters.
+      </div>
+      <div class="field-message" *ngIf="passwordStrengthError() == 'uppercaseError'">
+        Your password is too weak: It should contain a uppercase letter.
+      </div>
+      <div class="field-message" *ngIf="passwordStrengthError() == 'lowercaseError'">
+        Your password is too weak: It should contain a lowercase letter.
+      </div>
+      <div class="field-message" *ngIf="passwordStrengthError() == 'numericError'">
+        Your password is too weak: It should contain a numeric letter.
+      </div>
+
+      <mat-form-field>
+        <mat-label>Confirm your new password</mat-label>
+        <input
+          matInput
+          [type]="hide ? 'password' : 'text'"
+          formControlName="confirm"
+        />
+        <mat-icon
+          matSuffix
+          (mousedown)="hide = !hide"
+          [attr.aria-label]="'Hide password'"
+          [attr.aria-pressed]="hide"
+        >
+          {{ hide ? "visibility_off" : "visibility" }}
+        </mat-icon>
+      </mat-form-field>
+      <div class="field-message" *ngIf="matchingError()">
+        Passwords should be matching.
+      </div>
+
+      <button mat-raised-button color="primary">Submit</button>
```

Comment:
no event associated with the submit button

Reply:
fixed